webapp/channels/src/utils/constants.tsx

```diff
@@ -769,6 +769,11 @@ export const TELEMETRY_CATEGORIES = {
     REQUEST_BUSINESS_EMAIL: 'request_business_email',
     TRUE_UP_REVIEW: 'true_up_review',
     WORK_TEMPLATES: 'work_templates',
+    SYSTEM_ADMIN: 'system_admin',
+};
+
+export const TELEMETRY_EVENT = {
+    SYSTEM_CONSOLE_VISIT: 'system_console_visit',
```

Comment:
The naming convention followed is `*Action/Event Taken* + _ + *Object Taken Action On (optional - if any)* + _ + *Purpose/Result Of The Action*`. Based on that `click_system_console` looks like a more matching name. However, such event [already exists](https://mattermost.looker.com/dashboards/397?Event+Type+Or+Name=%22click_system_console%22&Source=&Category=). Any chance that the new event duplicates the existing one?